admin-dev/themes/new-theme/package.json

```diff
@@ -7,7 +7,9 @@
     "test": "./node_modules/.bin/mocha  --require babel-register --reporter spec \"tests/**/*.spec.js\"",
     "build": "NODE_ENV=production webpack --progress --colors --debug --display-chunks",
     "dev": "webpack --progress --colors --debug --display-chunks --watch",
-    "watch": "npm run dev"
+    "watch": "npm run dev",
+    "scss-lint": "stylelint \"**/*.scss\" --cache --cache-location .cache/.stylelintcache --formatter verbose",
+    "scss-fix": "stylelint \"**/*.scss\" --fix --cache --cache-location .cache/.stylelintcache"
```

Comment:
Should we target the scss directory directly?

Reply:
We can